db/schema.rb

```diff
@@ -33,6 +33,39 @@
   add_index "authentications", ["uid", "provider"], :name => "index_authentications_on_uid_and_provider", :unique => true
   add_index "authentications", ["user_id", "provider"], :name => "index_authentications_on_user_id_and_provider", :unique => true
 
+  create_table "c_rater_feedback_items", :force => true do |t|
```

Comment:
I think option #2 is fine as well.
I think it is also ok to wait on this until we have the runtime code done. At that point it might be more obvious if there is any benefit to unifying the feedback models. 

In my chat with Trudi, it seems the gating and 'feedback on feedback' will be more important, than the researcher report for this current block of work, so we might not need to worry about any of this  for a few months.